docs/src/installation_upgrade.md

```diff
@@ -228,19 +228,78 @@ When versions are not directly upgradable, the old version needs to be
 removed before installing the new one. This won't affect user data but
 only the operator itself.
 
-### Upgrading to 1.23.0, 1.22.3 or 1.21.5
+### Upgrading to 1.24.0 or 1.23.3
 
 !!! Important
     We encourage all existing users of CloudNativePG to upgrade to version
-    1.23.0 or at least to the latest stable version of the minor release you are
-    currently using (namely 1.22.2 or 1.21.4).
+    1.24.0 or at least to the latest stable version of the minor release you are
+    currently using (namely 1.23.3).
 
 !!! Warning
     Every time you are upgrading to a higher minor release, make sure you
     go through the release notes and upgrade instructions of all the
     intermediate minor releases. For example, if you want to move
-    from 1.21.x to 1.23, make sure you go through the release notes
-    and upgrade instructions for 1.22 and 1.23.
+    from 1.21.x to 1.24, make sure you go through the release notes
+    and upgrade instructions for 1.22, 1.23 and 1.24.
+
+#### From Replica Clusters to Distributed Topology
+
+One of the key enhancements in CloudNativePG 1.24.0 is the upgrade of the
+replica cluster feature.
+
+The former replica cluster feature, now referred to as the "Standalone Replica
+Cluster," is no longer recommended for Disaster Recovery (DR) and High
+Availability (HA) scenarios that span multiple Kubernetes clusters. Standalone
+replica clusters are best suited for read-only workloads, such as reporting,
+OLAP, or creating development environments with test data.
+
+For DR and HA purposes, CloudNativePG now introduces the Distributed Topology
+strategy for replica clusters. This new strategy allows you to build PostgreSQL
+clusters across private, public, hybrid, and multi-cloud environments, spanning
+multiple regions and potentially different cloud providers. It also provides an
+API to control the switchover operation, ensuring that only one cluster acts as
+the primary at any given time.
+
+This Distributed Topology strategy enhances resilience and scalability, making
+it a robust solution for modern, distributed applications that require high
+availability and disaster recovery capabilities across diverse infrastructure
+setups.
+
+You can seamlessly transition from a previous replica cluster configuration to
+a distributed topology by modifying all the `Cluster` resources involved in the
+distributed PostgreSQL setup. Ensure the following steps are taken:
+
+- The `externalClusters` section must be identical across all `Cluster`
+  resources in the distributed topology.
+- Each external cluster should include information enabling any other cluster
+  to recover from it as a source.
+- The current primary cluster should have the following content in the
+  `.spec.replica` stanza:
+```yaml
+replica:
+  enabled:
+  primary: <CLUSTER_NAME>
+  source: <SOURCE>
+```
```

Comment:
not sure this is helpful. We add it here but then don't explain what these things mean, and while `<CLUSTER_NAME>` is something all CNPG users will be familiar with, what is meant by `<SOURCE>`?
So, I think it's better to say less here and refer earlier to the replica cluster document.